### PR TITLE
Update Transcriptomics Reference-based tutorial

### PR DESCRIPTION
**Pull request by lldelisle**
This is a big update of this tuto before Smörgåsbord 2023.
- New gtf file: up to date + contig names match UCSC contrary to the previous one
- New STAR which allow to get coverage and gives a new way to get the strandness.
- Update all tool versions.

### CODE REVIEW

**Comment by lldelisle** [comment]
@bebatut would you mind to review my changes which are on the first part. Then I would need a new version of zenodo with the new gtf and the new counts (that I am generating now).

**Comment by lldelisle** [comment]
Is it ok if I put all tutorial authors as creators of workflows? and MIT license?

**Comment by lldelisle** [comment]
Just a note that I put here so it stays somewhere:

The big change in DEG numbers and GO and KEGG is due to the fact that in the previous version, the genes on contigs on the gtf were not transfered to the contigs on UCSC nomenclature, therefore while the reads were multimapped they were asigned to unique genes. In the new version, reads are multimapped and assigned to multiple genes. I checked that by applying the same ensembl->UCSC to the old version of the gtf (version 87) and I got similar results to the version 109 of the gtf.

**Comment by lldelisle** [comment]
The lint error is "This workflow has a creator but is missing an identifier for them. Please ensure all creators have valid ORCIDs."
See #4063

**Review by shiltemann — APPROVED** [review]
thanks! looks good from a technical point of view :+1:

@bebatut I know you're busy, but let us know if you would have some time to review this?

**Comment by lldelisle** [comment]
@mblue9 and @nomadscientist would you have time to review so I can record for Smorgasbord?

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 1]
```diff
@@ -182,11 +180,11 @@ We will first need to transform our the list of pairs to a simple list.
 >    >
 >    {: .question}
 >
->    As it is tidious to inspect all these reports individually we will combine them with {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.11+galaxy0) %}.
+>    As it is tidious to inspect all these reports individually we will combine them with {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.11+galaxy1) %}.
```
Comment (mblue9) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:183]:
```suggestion
>    As it is tedious to inspect all these reports individually we will combine them with {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.11+galaxy1) %}.
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 2]
```diff
@@ -673,7 +673,7 @@ Therefore we offer a parallel tutorial for the 2 methods which give very similar
 
 {% include _includes/cyoa-choices.html option1="featureCounts" option2="STAR" default="featureCounts" %}
```
Comment (mblue9) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:674]: Suggest moving this include line down to just above where the content changes so the learner can see what changes, "Counting reads per genes" section?

**Review by mblue9 — DISMISSED** [review]
Apologies, I'm a bit short on time for an in-depth review but from what I've seen the edits look great! I've added just a couple of suggestions for your consideration 😊

**Review by bebatut — DISMISSED** [review]
After quick scanning (sorry for the delay), the changes look good. Thanks for doing that!

File: topics/transcriptomics/tutorials/ref-based/tutorial.md (hunk 1 above)
Comment (lldelisle) [reply topics/transcriptomics/tutorials/ref-based/tutorial.md]: Thanks @mblue9

**Comment by lldelisle** [comment]
Thanks girls,
Would you mind to approve again (accepting @mblue9 suggestions cancelled your approval)?

**Review by bebatut — APPROVED** [review]

**Pull request merged**